### ISSUE TITLE
MongoDB error: “j must be numeric or a boolean value”

### ISSUE THREAD

**Issue by Jobarion** [bobthecow/genghis#203]
I was basically told to post this here since it's apperantly caused by genghis.
http://stackoverflow.com/questions/29757055/mongodb-j-must-be-numeric-or-a-boolean-value

**Comment by bobthecow** [comment]
- Are you using the Ruby or PHP version of Genghis?
- What Ruby/PHP version are you running?
- What MongoDB driver version are you running?
- How are you connecting to the database? (Is it on localhost? If not, what kind of URL are you entering?)

**Comment by Jobarion** [comment]
Ruby: 1.9.3p484
MongoDB: 3.0.2
I'm entering a domain. (e.g. mongo.example.net)

**Comment by mmeinert** [comment]
Same here
Ruby: ruby 2.1.5p273 (2014-11-13) [x86_64-linux-gnu]
MongoDB: 3.0.4
URL : IP_of_the_server:5678

Thank you in advance

**Comment by iby** [comment]
:+1:

**Comment by nriesco** [comment]
Same for me:
OS: MacOS X El Capitan - 10.11.1 (15B42) 
Ruby: 2.0.0p645 (2015-04-13 revision 50299) [universal.x86_64-darwin15]
MongoDB: 3.0.5
URL: (local) http://0.0.0.0:5678/
Browser: Chrome 46.0.2490.86 (64-bit)

Thanks

**Comment by matschaffer** [comment]
I haven't used genghis but I hit this exact error trying to use the 1.9.2 mongo gem to connect to a mongo 3 server in a rails app. Downgrading mongo to 2.6 resolved the error in that case.

**Comment by Jobarion** [comment]
Yes, it's due to the new authentication mechanism in MongoDB 3.0. I honestly doubt it's going to get fixed.
